status.c

```diff
@@ -121,6 +122,18 @@ static const char *status_format_str(char *buf, size_t buflen, size_t col, int c
         optional = 0;
       break;
 
+    case 'D':
+    {
+      struct Mailbox *mbox = Context ? Context->mailbox : NULL;
+      // If there's a description, use it. Otherwise, fall-through
+      if (mbox->desc)
+      {
+        mutt_str_strfcpy(tmp, mbox->desc, sizeof(tmp));
+        snprintf(fmt, sizeof(fmt), "%%%ss", prec);
+        snprintf(buf, buflen, fmt, tmp);
+        break;
+      }
+    }
```

Comment:
Please can you add a comment `/* fallthrough */` on a new line, after the `}`
That'll prevent Coverity worrying :-)